compiled_starters/git-starter-kotlin/your_git.sh

```diff
@@ -13,12 +13,11 @@ jarFile="$HOME/.cache/codecrafters-git-kotlin.jar"
 
 # Kotlin doesn't do incremental compilation, so let's hack this in ourselves.
 previousHash=$(cat "$cacheFile" 2>/dev/null || echo '')
-currentHash=$(tar -cP "$(dirname "$0")/app" | openssl sha1)
+currentHash=$(tar -c "$(dirname "$0")/app" | sha1sum)
```

Comment:
As mentioned in the commit `-P` isn't present in alpine tar and we don't need it (we can't silence the warning, though). OpenSSL is missing by default, but if it's really better we could install it, only ~1mb.